pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -100,6 +100,23 @@ type VSphereConfig struct {
 		VMName string `gcfg:"vm-name"`
 	}
```

Comment:
I guess you will remove all the unnecessary fields in Global fields later on? For example VMUUID, VMName etc. is unnecessary.

Remove VCenterIP, modify Datacenter -> Datacenters.